MacrobenchmarkSample/shared/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
<!--
```

Reply:
Must check the formatter, because it's the formatting setting for the project